faker/providers/person/es_CA/__init__.py

```diff
@@ -0,0 +1,107 @@
+# coding=utf-8
+from __future__ import unicode_literals
+from ..es_ES import Provider as PersonProvider
+
+
+class Provider(PersonProvider):
+    """
+    Adds popular Catalan names.
+    https://www.idescat.cat/pub/?id=aec&n=946&lang=es&t=2018
+    https://www.idescat.cat/pub/?id=aec&n=947&lang=es&t=2018
+    """
+    first_names_male = (
+        'Adam',
+        'Albert',
+        'Aleix',
+        'Àlex',
+        'Álex',
```

Comment:
this two names are treated like one and the same in the original dataset. Including them both would result in having them picked twice as often. I'd just toss a coin and pick one, unless there's one deemed more 'canonical'.

Reply:
Another good point. Amended.